DEVELOPING.md

```diff
@@ -13,7 +13,9 @@ You will need:
 
 Komga's commit messages follow the [Conventional Commits](https://www.conventionalcommits.org/) standard. This enables automatic versioning, releases, and release notes generation.
 
-Commit messages are enforced using commit hooks ran on the developer's PC. To install the necessary tooling, you need to run `npm install` in the root folder of the project. This will install the necessary commit hooks.
+Commit messages are enforced using commit hooks ran on the developer's PC. To install the necessary tooling, you need to run `npm install` in the root folder of the project. This will install the necessary commit hooks. You must then run `npm install` again within the `komga-webui` folder to get the tools the commit hooks use.
```

Comment:
This is incorrect. The tooling is only in the root folder. What's in `komga-webui` is only for the webui.

Reply:
The commit hook calls the vue-cli-service program, which isn't included in the root package.json, it's only in the komga-webui/package.json, so I had to install those packages before I was able to commit.

It does look like it'd possible to commit from root after installing those packages though, I'll remove that part

Reply:
I think it would be more explicit to add a section on what to run when setting the project for the first time. You can omit those changes altogether, i will add a new section entirely in the file.

Reply:
Removed